lib/membrane-element-rtp-aac/depayloader.ex

```diff
@@ -1,20 +1,10 @@
 defmodule Membrane.RTP.AAC.Depayloader do
   use Membrane.Filter
   alias Membrane.Buffer
-  alias Membrane.AAC
+  alias Membrane.{AAC, RemoteStream, RTP}
 
-  def_input_pad :input, demand_unit: :buffers, caps: :any
-  def_output_pad :output, caps: {AAC, encapsulation: :none}
-
-  def_options profile: [
-                default: :LC
-              ],
-              sample_rate: [
-                default: 44100
-              ],
-              channels: [
-                default: 2
-              ]
+  def_input_pad :input, demand_unit: :buffers, caps: RTP
+  def_output_pad :output, caps: {RemoteStream, content_format: AAC, type: :packetized}
 
   @impl true
   def handle_init(options) do
```

Comment:
Shouldn't we also change
`{:ok, options |> Map.merge(%{leftover: <<>>})}`
to
`{:ok, %{}}`?

I don't see it is used anywhere.